code/game/objects/items/melee/energy.dm

```diff
@@ -228,3 +228,65 @@
 	desc = "An extremely sharp blade made out of hard light. Packs quite a punch."
 	icon_state = "lightblade"
 	item_state = "lightblade"
+
+/obj/item/weapon/melee/transforming/energy/sword/bikehorn

```

Comment:
change all /obj/item/weapon/melee/transforming/energy/sword/bikehorn to /obj/item/melee/transforming/energy/sword/bikehorn